Packs/CortexXDR/TestPlaybooks/Test_Playbook_-_Cortex_XDR_Malware_-_Incident_Enrichment.yml

```diff
@@ -0,0 +1,4565 @@
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "168"
+      Verified:
+      - "136"
+    separatecontext: false
+    conditions:
+    - label: Verified
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: incident
+                accessor: accountmemberof
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 3350,
+          "y": -380
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "162":
+    id: "162"
+    taskid: 54a9b87c-6cb1-412c-8523-6c2c7dc0959d
+    type: condition
+    task:
+      id: 54a9b87c-6cb1-412c-8523-6c2c7dc0959d
+      version: -1
+      name: Verify Account Name
+      description: |
+        Verify that the ‘accountname’ incident field was filled out correctly.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "169"
+      Verified:
+      - "136"
+    separatecontext: false
+    conditions:
+    - label: Verified
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: incident
+                accessor: accountname
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 3750,
+          "y": -380
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "163":
+    id: "163"
+    taskid: 347ebfb0-b78b-4dcf-8595-e86f7234e8a0
+    type: condition
+    task:
+      id: 347ebfb0-b78b-4dcf-8595-e86f7234e8a0
+      version: -1
+      name: Verify Account Status
+      description: |
+        Verify that the ‘accountstatus’ incident field was filled out correctly.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "170"
+      Verified:
+      - "136"
+    separatecontext: false
+    conditions:
+    - label: Verified
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: incident
+                accessor: accountstatus
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 4150,
+          "y": -380
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "164":
```

Comment:
can we move this task and other "account" context key checks to the Account section below?
![image](https://user-images.githubusercontent.com/88268646/235606623-80f1b28a-fd99-4ec8-8ee1-0c32a7c3a43a.png)

Reply:
Removed the task from the 'check incident fields'